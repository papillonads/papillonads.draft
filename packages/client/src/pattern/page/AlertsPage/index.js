import AsyncAlertsPage from './AsyncAlertsPage'

export default AsyncAlertsPage
