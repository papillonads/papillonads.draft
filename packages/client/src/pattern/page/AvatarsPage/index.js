import AsyncAvatarsPage from './AsyncAvatarsPage'

export default AsyncAvatarsPage
