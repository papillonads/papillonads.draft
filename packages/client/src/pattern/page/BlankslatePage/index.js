import AsyncBlankslatePage from './AsyncBlankslatePage'

export default AsyncBlankslatePage
