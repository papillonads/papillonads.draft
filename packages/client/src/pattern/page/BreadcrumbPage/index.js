import AsyncBreadcrumbPage from './AsyncBreadcrumbPage'

export default AsyncBreadcrumbPage
