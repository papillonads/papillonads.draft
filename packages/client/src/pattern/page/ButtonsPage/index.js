import AsyncButtonsPage from './AsyncButtonsPage'

export default AsyncButtonsPage
