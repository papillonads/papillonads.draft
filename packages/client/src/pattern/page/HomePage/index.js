import AsyncHomePage from './AsyncHomePage'

export default AsyncHomePage
