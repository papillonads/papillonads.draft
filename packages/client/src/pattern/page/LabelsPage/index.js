import AsyncLabelsPage from './AsyncLabelsPage'

export default AsyncLabelsPage
