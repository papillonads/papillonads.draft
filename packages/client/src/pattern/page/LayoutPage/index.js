import AsyncLayoutPage from './AsyncLayoutPage'

export default AsyncLayoutPage
