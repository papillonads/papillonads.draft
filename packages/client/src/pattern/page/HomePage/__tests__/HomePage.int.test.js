import React from 'react'
import { asyncRender, defaultRender } from './HomePage.int.render'
import { list } from '../../../../../../server/src/data/page/pages'

jest.mock('@apollo/react-hooks', () => ({ useQuery: () => {} }))
jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

const apolloReactHooksMockObject = require('@apollo/react-hooks')

describe('<HomePage />', () => {
  const pagesDataObject = {
    data: {
      pages: {
        list,
      },
    },
  }

  afterEach(() => jest.clearAllMocks())

  describe('Snaphot', () => {
    test('must match asyncRender', () => {
      jest.spyOn(apolloReactHooksMockObject, 'useQuery').mockReturnValue(pagesDataObject)
      expect(global.renderToJSON(asyncRender)).toMatchSnapshot()
    })

    // test('must match defaultRender', () => {
    //   jest.spyOn(apolloReactHooksMockObject, 'useQuery').mockReturnValue(pagesDataObject)
    //   expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    // })

    test('must match homePageNull', () => {
      jest.spyOn(apolloReactHooksMockObject, 'useQuery').mockReturnValue({})
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
