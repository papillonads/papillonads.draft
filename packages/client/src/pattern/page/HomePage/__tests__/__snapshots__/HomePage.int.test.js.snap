// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Snaphot must match asyncRender 1`] = `null`;

exports[`<HomePage /> Snaphot must match homePageNull 1`] = `null`;
